### ISSUE TITLE
gosu doesn't work

### ISSUE THREAD

**Issue by derekslenk** [x4121/docker-taskd#1]
gosu produces constant errors about not being able to find files. I have no idea if this is active though

**Comment by x4121** [comment]
Hi, thanks for the info. Have to admit that I stopped using taskd a long while ago. I can try to have a look at the problem on the weekend, but I'd be glad if someone could look into it.

**Comment by derekslenk** [comment]
> 
> 
> Hi, thanks for the info. Have to admit that I stopped using taskd a long while ago. I can try to have a look at the problem on the weekend, but I'd be glad if someone could look into it.

Did you find something else you like better? Curious myself as I am not totally sure I like it

**Comment by x4121** [comment]
> Did you find something else you like better? Curious myself as I am not totally sure I like it

Not really, main purpose for the sync with taskd was using taskwarrior on my smartphone, but this didn't work at all and most of the time resulted in a broken state (on the phone). I don't know if the smartphone app (which was not developed by the taskwarrior team) has these bugs fixed or if there is a better solution now.
Currently I just use a Nextcloud folder to sync the tasks (since I only use it for myself and never use multiple computers at a time, this shouldn't lead to conflicts), since it's much less complexity to manage.

So I still love taskwarrior, I'm just not using the server anymore.

**Comment by x4121** [comment]
Hi @derekslenk,
sorry, didn't have time to do it on the weekend. I just updated the base image and version of gosu and am not getting errors when running commands with gosu.
Can you check if this fixes the issue you had?
